src/main/java/io/vertx/core/http/HttpServerOptions.java

```diff
@@ -280,6 +281,13 @@ public HttpServerOptions addEnabledCipherSuite(String suite) {
   }
 
   @Override
+  @GenIgnore
```

Comment:
I believe we should try to handle Map<String, Options> in codegen to not have GenIgnore